consensus/module.go

```diff
@@ -58,7 +65,18 @@ type consensusModule struct {
 	MaxBlockBytes uint64
 }
 
-func Create(cfg *genesis.Config, genesis *genesis.GenesisState) (modules.ConsensusModule, error) {
+func Create(configPath, genesisPath string, useRandomPK bool) (modules.ConsensusModule, error) {
```

Comment:
I would like for us to discuss the addition of `useRandomPK` before moving forward? Seems like a big step back in our modular architecture.

In the previous approach where we could "inject" a random private key without hacks like this shows that it was a good design from a "separation of concerns" an good for testing/mocking/development. I understand it doesn't satisfy the "encapsulation requirement", but just wanted to point that the whenever we need to start adding params like "useRandomPk", it's often a sign that there might be a better way of doing something.

Ccing @adshmh and @deblasis for context so we can discuss it as a group and ideate.

Reply:
Injecting a private key is a `hack` the real solution is to use a different configuration file for the `debug_client`